modules/menu/menu.c

```diff
@@ -655,23 +655,34 @@ struct ua   *menu_ua_carg(struct re_printf *pf, const struct cmd_arg *carg,
 		struct pl *word1, struct pl *word2)
 {
 	int err;
-	struct le *le;
+	struct le *le = NULL;
 	uint32_t i;
 	struct ua *ua = carg->data;
 
 	if (ua)
 		return ua;
 
-	err = re_regex(carg->prm, str_len(carg->prm), "[^ ]+ [^ ]+", word1,
-			word2);
+	err = re_regex(carg->prm, str_len(carg->prm), "[^ ]+[ ]*[0-9]*",
+		       word1, NULL, word2);
```

Comment:
Ok, looks better than the other PR. Maybe we should remove the pointer `word2 ` in the parameter list of this function, because it is not needed.

Reply:
in re_regex a start (*) means optional (zero or more matches).

in any case the `struct pl` parameter will be initialized to PL_INIT or the match

Reply:
> in re_regex a start (*) means optional (zero or more matches).

Yes, I know.

> in any case the struct pl parameter will be initialized to PL_INIT or the match

Good to know.

 But what I mean is that the parameter `word2` in function `menu_ua_carg()` is never used. Ok, it is not really related to this PR.